supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,95 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+=== Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links.
++
+Example JavaScript:
+----
+<script type="text/javascript" src="/path/to/script.js"></script>
+----
+* Do not use Flash links.
++
+Example Flash:
++
+----
+on(release)
+{
+getURL("http://www.example.com", "_blank");
+}
+----
+* Do not use links in Frames or iframes.
++
+Example iFrame:
++
+----
+ <iframe src="url" title="description"></iframe> 
+----
+
+=== Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location.
+* Use a concise sentence or sentence fragment as the link text.
+* Do not use bare URLs. Bare URLs are unhelpful because they do not provide adequate context about the link target.
+* Avoid irrelevant link text.
+
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within \www.redhat.com.
+
+.Example AsciiDoc: Internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to another guide or to a user story within the same guide:
```

Comment:
I don't understand the "or to a user story within the same guide". If it's in the same guide, shouldn't an xref still be used?